doc/source/whatsnew/v1.1.3.rst

```diff
@@ -22,7 +22,7 @@ Fixed regressions
 
 Bug fixes
 ~~~~~~~~~
--
+- Bug in :meth:`isin()` when using NaN and a row length above 1,000,000 (:issue:`22205`)
```

Comment:
```suggestion
- Bug in :meth:`Series.isin` when using ``NaN`` and a row length above 1,000,000 (:issue:`22205`)
```

Reply:
This in theory also for Dataframe.isin, I could see `isin` documented like this previously, which one is the prefered way?

Reply:
Could also mention both with :meth:\`Series.isin\` and :meth:\`DataFrame.isin\`

Reply:
prefer the specific ones (Series/DataFrame)